extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/channels.xml

```diff
@@ -31,9 +31,12 @@
 	</channel-type>
 
 	<channel-type id="clearqueue" advanced="true">
-		<item-type>Switch</item-type>
+		<item-type>String</item-type>
```

Comment:
This change can break existing setups.

Reply:
thanks, this was not intended is not even necessary. Switch items will also get the push buttons.